docs/documentation.md

```diff
@@ -7,20 +7,21 @@
 
```

Comment:
`README.md` has extra [paragraph](https://github.com/example42/psick/tree/production/docs#puppet-control-repo-documentation) but I do not know the purpose of `documentation.md` so I synced only same paragraphs in both pages.